pybombs-commondeps/Dockerfile

```diff
@@ -1,16 +1,31 @@
-FROM pybombs/pybombs-prefix:2.3.3
+FROM pybombs/pybombs-prefix:2.3.4
 LABEL maintainer=martin@gnuradio.org
 
 # This allows setting the makewidth temporarily to a higher value
 ARG makewidth=2
 
+# Temporarily set frontend to noninteractive to avoid warnings
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Ubuntu 20.04 gets stuck on tzdata configuration during build.
+# Providing a default timezone that can be overridden using build-args:
+ARG TZ=Etc/UTC
+RUN ln -fs /usr/share/zoneinfo/$TZ /etc/localtime
+
 RUN pybombs config makewidth $makewidth
+
+# installing qwt5 will fail due to lack of X11 headers.
+# Temp fix until this is solved in the pybombs recipe (see gnuradio/gr-recipes#197):
+RUN apt-get update && apt-get install -y \
+	libqwt-qt5-dev \
+	python3-pyqt5.qwt
```

Comment:
Echoing my comments from https://github.com/gnuradio/gr-recipes/issues/197: Is qwt5 support in Ubuntu 20.04 important? GNU Radio 3.8 uses qwt6.

Reply:
Ok, I wasn't aware 3.8 was firmly on qwt6 - will adapt the Dockerfile and see if that builds, that would get rid of the workaround.